gui-wm/wayfire/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>gentoo@aisha.cc</email>
+        <name>Aisha Tammy</name>
+    </maintainer>
+    <maintainer type="project">
+        <email>proxy-maint@gentoo.org</email>
+        <name>Proxy Maintainers</name>
+    </maintainer>
+    <use>
+      <flag name='wcm'>Pulls in wayfire config manager.</flag>
+      <flag name='wf-shell'>Pulls in wf-shell (recommended, until a full fledged DE is made).</flag>
+      <flag name='wf-config'>Makes wayfire use the system-wide wf-config, else builds it as a submodule. DO NOT DISABLE UNLESS YOU KNOW WHAT YOU ARE DOING.</flag>
```

Comment:
I wonder if it's worth exposing these for now if they're not useful for the time being / act as a footgun.

Reply:
They are unfortunately needed when being tested for stability of the base software when it pull certain subprojects.

I have actually disabled a couple of other options, related to graphics settings, which I am testing right now. I will add them later after the simple ebuild has worked out first.